src/pages/registrering/Spørsmål.ts

```diff
@@ -0,0 +1,15 @@
+import { Behov } from '../../types/Behov';
+
+export interface Svaralternativ<T> {
+    value: T;
```

Comment:
`value` blir vel brukt som nøkkel til et svar. Så kanskje vi kunne kalt feltet for `nøkkel` eller noe lignende som er mer beskrivende. value er jo ganske generisk